.lintstagedrc

```diff
@@ -1,8 +1,11 @@
 {
-  "app/**/*.{js,jsx,json,css,scss,graphql,gql}": [
+  "app/**/*.{js,jsx,css,scss,json,graphql,gql}": [
     "prettier --write",
   ],
-  "api/**/*.{js,jsx,json,css,scss,graphql,gql}": [
+  "api/**/*.{js,jsx,css,scss,json,graphql,gql}": [
+    "prettier --write",
+  ],
+  "app/styles/**/!*.{css,scss}": [
```

Comment:
Remove this file from diff, I don't see any changes.

Reply:
Done